cmdLine/src/main/scala/topgun/cmdline/FileParser.scala

```diff
@@ -52,8 +47,10 @@ class FileParser(file: File, cmdLine: JfrParseCommandLine, totals: Totals, confi
 
   var tlabSize = 2048L
 
-  def handleTlabConfiguration(event: IEvent): Unit = {
+//  OUR JFR FILE NEVER INVOKED THIS METHOD. WE CANNOT CONFIRM IF THE VALUE'S NAME HAS BEEN CHANGED IN THE JAVA 11 APIS { 'usesTLABs' , 'minTLABSize' }
```

Comment:
record a very small file and look at the events in there. It could well have changed name